kolibri/core/assets/src/core-actions.js

```diff
@@ -349,6 +443,85 @@ function stopTrackingProgress(store, Kolibri) {
   updateTimeSpent(store, Kolibri, true);
 }
 
+function saveMasteryLog(store, Kolibri) {
```

Comment:
Possibly not, although there may be independent changes to an attemptlog (additional answers after the first incorrect answer) that have no effect on the masteryLog, however, we might just let the dirty checking take care of that.